ignite/cmd/scaffold_chain_registry.go

```diff
@@ -0,0 +1,68 @@
+package ignitecmd
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/ignite/cli/v29/ignite/pkg/cliui"
+	"github.com/ignite/cli/v29/ignite/services/chain"
+	"github.com/ignite/cli/v29/ignite/services/scaffolder"
+)
+
+// NewScaffoldChainRegistry returns the command to scaffold the chain registry chain.json and assets.json files.
+func NewScaffoldChainRegistry() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "chain-registry",
+		Short: "Configs for the chain registry",
+		Long: `Scaffold the chain registry chain.json and assets.json files.
+
```

Comment:
Should certain fields should be configurable? 

For instance:

```
c.Flags().String("status", "upcoming", "Chain status (mainnet, testnet, upcoming, etc.)")
```

and then consumed in `scaffoldChainRegistryFiles`

```
status, _ := cmd.Flags().GetString("status")
```

And passed down to `AddChainRegistryFiles`

```
if err = sc.AddChainRegistryFiles(c, cfg, status); err != nil {
    return err
}
``

Reply:
I wasn't super sure. As youd scaffold this when you need the frontend, and usually it is at devnet. Wanted to keep it KISS. But we can always add if you find it valuable.